db/migrate/20170613144708_add_sti_to_container_image.rb

```diff
@@ -1,6 +1,5 @@
 class AddStiToContainerImage < ActiveRecord::Migration[5.0]
   def change
     add_column :container_images, :type, :string
-    add_column :container_images, :ems_ref, :string
```

Comment:
Please add a new migration @enoodle. 
That way developers that already ran the migration will remove the new column